app/controllers/platform/api/v1/accounts_controller.rb

```diff
@@ -3,16 +3,12 @@ def create
     @resource = Account.new(account_params)
     @resource.save!
     @platform_app.platform_app_permissibles.find_or_create_by(permissible: @resource)
-    render json: @resource
   end
 
-  def show
-    render json: @resource
-  end
+  def show; end
 
   def update
```

Comment:
@smartdev58  could you also add a spec for the update endpoint since this change can impact account update  API calls? We want to ensure that existing features aren't disabled if someone makes a patch request to the account endpoint without specifying the enabled feature flags.

Reply:
@sojan-official Thank you for your review. I will add the spec for update endpoint.

Reply:
@sojan-official I have added the required spec. Please review